src/components/List/_list.scss

```diff
@@ -10,7 +10,6 @@
   }
   &--content {
     flex: 1 1 0 0;
-    background: $ui-01;
```

Comment:
I am adding this back as this leaves a blank space in list that are not large enough 
![image](https://user-images.githubusercontent.com/1590966/91890165-0b110580-ec5d-11ea-926c-2aef2629e41d.png)